changes/2338.feature.md

```diff
@@ -0,0 +1 @@
+Add AWEBP support for asset URLs, increase sticker asset availability, & prepare various properties for deprecation.
```

Comment:
The deprecations should be listed in a different changelog fragment called `2338.deprecations.md`